components/ArticleDetails.jsx

```diff
@@ -1,29 +1,61 @@
 import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
 import { fetchArticlesById } from "../api";
+import { getCommentbyId } from "../api";
+import { CommentCard } from "./commentCard";
 
 export function ArticleDetails() {
   const { article_id } = useParams();
   const [article, setArticle] = useState(null);
   const [isLoading, setIsLoading] = useState(true);
+  const [comments, setComments] = useState([]);
 
   useEffect(() => {
     setIsLoading(true);
     fetchArticlesById(article_id).then((articleData) => {
+      console.log(articleData);
       setArticle(articleData);
       setIsLoading(false);
     });
   }, [article_id]);
 
-  if (isLoading) return <p>Loading...</p>;
+  useEffect(() => {
```

Comment:
I would steer clear of more than one use effect in a component. You can handle both of these api calls in one useEffect. By having more than one effect you increase the risk of unhandled re renders.